src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -1042,11 +1042,19 @@ private static GeneratableElement getExampleFor(ModElement modElement, Random ra
 
 					block.boundingBoxes.add(box);
 				}
+			} else {

```

Comment:
Currently tx blocks are not tested because "mess" like this is needed in order for them to operate due to constraints (eg. no gravity support). So for now omit this test, we need to work on a full support of tx blocks in tests in the future.